src/abi.h

```diff
@@ -45,6 +74,12 @@ typedef struct dhcp_overrides {
     guint metric;
 } NetplanDHCPOverrides;
 
+typedef struct ipv6_overrides {
```

Comment:
nitpick (non-blocking): This struct should probably be called `ra_overrides`. But it's only an internal implementation detail, we can change it at any time.